src/KafkaFlow.SchemaRegistry/KafkaFlow.SchemaRegistry.csproj

```diff
@@ -9,7 +9,6 @@
 
     <ItemGroup>
         <PackageReference Include="Confluent.SchemaRegistry" Version="2.1.1" />
-        <PackageReference Include="Microsoft.IO.RecyclableMemoryStream" Version="2.3.2" />
```

Comment:
I think @filipeesch can shed some light here, if this is necessary or not.

Reply:
https://github.com/Farfetch/kafkaflow/pull/486#discussion_r1422258927